kale/embed/mpca.py

```diff
@@ -93,38 +129,57 @@ def fit(self, X):
         return self
 
     def transform(self, X):
-        """
-        Parameter:
-            X: array-like, shape (I1, I2, ..., n_samples)
-        ----------
-        Return:
-            Transformed data, array-like, shape (i1, i2, ..., n_samples)
+        """Perform dimension reduction on X
+
+        Args:
+            X (ndarray): shape (I1, I2, ..., n_samples)
+
+        Returns:
+            ndarray: Transformed data, shape (i1, i2, ..., n_samples)
```

Comment:
Is there a variable name?
I seem to understand why. If this is what sklearn does, it is fine.